runtime/manifest-parser.peg

```diff
@@ -156,6 +176,16 @@ RecipeView
     }
   }
 
+RecipeSlot
+  = "renders" ref:(whiteSpace ViewRef)? name:(whiteSpace Name)? eolWhiteSpace
```

Comment:
this should be `lowerIdent` (for a local name) rather than `ViewRef` (ViewRef was only intended to be used when mapping views), although that might cause some problems with 'renders as x'